MatrixSDK/MXRestClient.h

```diff
@@ -2122,13 +2122,16 @@ Get the maximum size a media upload can be in bytes.
 
  @param deviceKeys the device keys to send.
  @param oneTimeKeys the one-time keys to send.
+ @param fallbackKeys the fallback keys to send.
 
  @param success A block object called when the operation succeeds.
  @param failure A block object called when the operation fails.
 
  @return a MXHTTPOperation instance.
  */
-- (MXHTTPOperation*)uploadKeys:(NSDictionary*)deviceKeys oneTimeKeys:(NSDictionary*)oneTimeKeys
```

Comment:
Should this old method remain for API compatibility?

Reply:
I thought about it but seeing as we don't have separate methods for uploading keys or oneTimeKeys I figured this is more straight forward.